service-worker.js

```diff
@@ -0,0 +1,19 @@
+---
+---
+
+importScripts('/scripts/sw-toolbox.js');
+
+var swOptions = {
+  debug: true
+}
+
+var preCachedRessources = [
+  '{{ "/css/main.css" | prepend: site.baseurl }}?t={{ site.time | date_to_xmlschema }}"',
+  {% for page in site.pages %}
+  '{{ page.url }}',
+  {% endfor %}
+];
+
+toolbox.precache(preCachedRessources);
+
+toolbox.router.get('/(.*)', toolbox.cacheFirst, swOptions);
```

Comment:
We could do networkOnly, but @xtuc wanted that the user be able to search even when offline - as mostly it is just static data updated via commits. So, it could be networkFirst with a separate cache name and maxEntries.